fb_dataset.py

```diff
@@ -82,7 +82,7 @@
 
 
 #### TRAIN THE MODEL ####
-# Build a simple regression model
+# Build a regression model
```

Comment:
the change in the model looks good to me! thanks.

I can only comment on new chanegs to the code, so I'll add my comments about the other parts of the code here:

1. Let's start with the simplest model: just the features from LDA600, no demographics or countries. This will avoid the problem of missing data and having to impute it, and the problems with data types (e.g., timestamps). 
2. On top of the simplest model, you can use that one that you had created (using more than just the LDA600 features). I see that you repeated the data processing steps manually for the train and test splits. Please write a function to do all the preprocessing - this makes your code cleaner and less prone to error.
3. Don't use the test.csv. Use only the train.csv. We'll use the test.csv later. For now, assume that the train.csv is all you have - how do you train and evaluate a model based on just this dataset - you would have to create your own data partitions - remember to set the random seed so we can reproduce the partitions. 
4. Use a function to take in multiple inputs and convert them to tensors.
5. Can you at the end show a table/dataframe or just a simple print with: the baseline MSE (if the model predicts the mean all the time), the MSE of the models with and without lasso, with and without the different features.